### ISSUE TITLE
Not possible to adjust sliders in small steps on Layer Model Screen

### ISSUE THREAD

**Issue by Nancy-Salpepi** [phetsims/greenhouse-effect#261]
**Test device**
MacBook Air (m1 chip)

**Operating System**
13.1

**Browser**
safari/chrome

**Problem description**
For https://github.com/phetsims/qa/issues/885, on the Layer Model Screen, it isn't possible to adjust any of the sliders in small steps in the Sunlight and Infrared panels. Using shift + arrow moves the slider to the next tick mark (same as using only the arrows). 
Using shift + arrow works correctly for Flux Meter and Greenhouse Gas Concentration slider.

**Steps to reproduce**
1. On the Layer Model Screen, tab to any slider
2. Use shift + arrow to adjust

<details>
Troubleshooting information:
!!!!! DO NOT EDIT !!!!!
Name: ‪Greenhouse Effect‬
URL: https://phet-dev.colorado.edu/html/greenhouse-effect/1.1.0-rc.2/phet/greenhouse-effect_all_phet.html
Version: 1.1.0-rc.2 2023-01-18 00:39:07 UTC
Features missing: applicationcache, applicationcache, touch
Flags: pixelRatioScaling
User Agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/605.1.15 (KHTML, like Gecko) Version/16.2 Safari/605.1.15
Language: en-US
Window: 1221x710
Pixel Ratio: 2/1
WebGL: WebGL 1.0
GLSL: WebGL GLSL ES 1.0 (1.0)
Vendor: WebKit (WebKit WebGL)
Vertex: attribs: 16 varying: 30 uniform: 1024
Texture: size: 16384 imageUnits: 16 (vertex: 16, combined: 32)
Max viewport: 16384x16384
OES_texture_float: true
Dependencies JSON: {}
</details>

**Comment by arouinfar** [comment]
Thanks @Nancy-Salpepi. @jbphet there isn't anything we need to do here for Greenhouse.

> on the Layer Model Screen, it isn't possible to adjust any of the sliders in small steps in the Sunlight and Infrared panels. Using shift + arrow moves the slider to the next tick mark (same as using only the arrows).

The sliders in the Sunlight & Infrared panels are all discrete, with values restricted to the tick marks. The standard step size is a single tick mark, so subdividing further is unhelpful. We've run into this before in Fourier: Making Waves, see https://github.com/phetsims/fourier-making-waves/issues/167#issuecomment-909477117.

Discrete sliders generally have ~10 (or fewer) tick marks, which is roughly equivalent to the number of standard steps it takes to move a slider through its full range. There isn't any real need for supporting a smaller step size, and doing so would almost certainly allow users to access values that were never intended.  

Since the Layer Model screen does have another slider that supports the shift modifier, the Keyboard Shortcuts dialog content is accurate. However, I wonder what happens in a sim where the sliders do not support smaller/larger steps. Are those sections of the dialog optional @jessegreenberg?

@Nancy-Salpepi should the QA book be updated to make a note that smaller steps are often irrelevant for discrete sliders?

**Comment by Nancy-Salpepi** [comment]
>@Nancy-Salpepi should the QA book be updated to make a note that smaller steps are often irrelevant for discrete sliders?

Sure I can add that in right now @arouinfar. I only tested it because it was listed in the Keyboard Shortcuts Dialog, but now I know that doesn't apply to discrete sliders 🙂.

**Comment by jessegreenberg** [comment]
No, there is no option to remove that row currently. We could add that though if you would like. In this case though the shift key still applies to the flux meter sensor so keeping it for this sim may make sense. Let me know if there should be an option for that.

**Comment by arouinfar** [comment]
> In this case though the shift key still applies to the flux meter sensor so keeping it for this sim may make sense. 

Agreed, no changes are necessary for Greenhouse Effect.

> Let me know if there should be an option for that.

Thanks @jessegreenberg. At this stage, it's a fairly hypothetical case. I think we can wait to add the option until we get to a sim where the smaller and/or larger step modifiers are irrelevant, but I'll leave it up to you to decide.

**Comment by pixelzoom** [comment]
It looks the decision was to do nothing for Greenhouse.  That the remaining work was to create a general issue like https://github.com/phetsims/scenery-phet/issues/811, so I did so. 

So... Closing. If I missed anything, please reopen.
